.github/workflows/build.yml

```
@@ -73,6 +73,8 @@ jobs:
               -DMAD=ON
               -DMEDIAFOUNDATION=ON
               -DSTATIC_DEPS=ON
+              -DCODESIGN=ON
```

```suggestion
              -DWINDOWS_CODESIGN=ON
```
Please rename this to avoid confusion with the macOS code signing stuff.